### PR TITLE
chore: Spotless formatter

### PR DESCRIPTION
**Pull request by MarcinVaadin**
Enabled spotless formatter + fixed some code related minor warnings

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -56,6 +57,19 @@ dependencies {
     }
 }
 
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    kotlin {
+        // by default the target is every '.kt' and '.kts` file in the java sourcesets
+        ktfmt("0.51").googleStyle().configure {
```
Comment (Artur-) [inline build.gradle.kts:63]: Let's not use different configuration than in the other projects. Here is the config from the starter https://github.com/vaadin/skeleton-starter-flow-spring/blob/7dc62ba6e702576bae7346361d7070f4e04fb2df/pom.xml#L100-L125

Comment (Artur-) [reply build.gradle.kts:63]: or does palantir not support kotlin files?

**Review by Artur- — APPROVED** [review]
otherwise looks good

File: build.gradle.kts (hunk 1 above)
Comment (MarcinVaadin) [reply build.gradle.kts:63]: I think so, I can't find palantir for Kotlin :-/

**Review by Artur- — APPROVED** [review]

**Pull request merged**